packages/runtime/datastore-definitions/src/dataStoreRuntime.ts

```diff
@@ -52,7 +82,7 @@ export interface IFluidDataStoreRuntime
 	// eslint-disable-next-line @typescript-eslint/no-explicit-any
 	readonly options: Record<string | number, any>;
 
-	readonly deltaManager: IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>;
+	readonly deltaManagerErased: IDeltaManagerErased;
```

Comment:
This is the main change in this PR: everything else is in response to this.

Reply:
Since we need a separate name for the public type erased member and the actual internal one with how the type erasing is implemented, I renamed this one as that involved less code changes. I'd like more opinions on if I should have renamed the internal one instead and left this as `deltaManager`.

Reply:
I see the @sealed being added in this PR, but as long as this is public this seems worth a changeset? I'm also not aware if we've come to consensus on "we can break @sealed stuff because consumers _shouldn't_ be extending/implementing it even if nothing technically stops them".

About renaming this vs the internal one, my gut feeling would have been to keep the public one as `deltaManager`, but... not sure consumers should ever use `IFluidDataStoreRuntime.<this property>`, but if they do I think in their shoes I'd find it a weird name for a property.

Reply:
oh yes, this absolutely needs a changeset. I also agree about the rename, I was mostly hoping to get some confirmation this is a good approached before doing that work. I think I have that now, so I should go do that work!

Reply:
Changeset added rename done.